frontend/packages/dev-console/src/utils/application-utils.ts

```diff
@@ -110,7 +110,7 @@ export const updateResourceApplication = (
       _.forEach(list, (item) => {
         // verify the case of no previous application
         if (prevApplication || !_.get(item, ['metadata', 'labels', 'app.kubernetes.io/part-of'])) {
-          patches.push(updateItemAppLabel(item.resourceKind, item, application));
+          patches.push(updateItemAppLabel(modelFor(item.kind), item, application));
```

Comment:
I copied the logic when it calls this function to get the resource kind:
https://github.com/openshift/console/pull/2418/files#diff-c8b822bac394ac5898fa58d8b3cd609bR435

Reply:
👍  Thanks @andrewballantyne, just have gotten mixed up during the rebase.